packages/perseus-editor/src/__stories__/flags-for-api-options.ts

```diff
@@ -6,6 +6,9 @@ export const flags = {
         circle: true,
         quadratic: true,
         sinusoid: true,
```

Comment:
While this isn't strictly part of the bug fix, I think it makes sense to turn these graphs "on" so that we can test them within the storybook pages for Mafs graphs.